docs/slurk_bots_events.rst

```diff
@@ -0,0 +1,228 @@
+.. _slurk_bots_events:
+
+=========================================
+Bot related Events
+=========================================
+
+Rooms
+~~~~~
+There are two types of events indicating that a new room was created. The first of the two ``new_room`` is the more general one. It can be triggered for every created room independently of its purpose and layout. Both events are received by all connected users.

```

Comment:
```suggestion
There are two types of events showing that a new room was created. The first of the two ``new_room`` is the more general one. It can be triggered for every created room, independently of its purpose and layout. Both events are received by all connected users.
```